agent/worker.go

```diff
@@ -142,14 +143,21 @@ func (w *worker) Assign(ctx context.Context, assignments []*api.AssignmentChange
 		return err
```

Comment:
Non-blocking:  there is a comment on L140 that says that secrets need to be updated before tasks - can we update this to reflect that the same is true for configs?